wpilib/tests/test_shuffleboard.py

```diff
@@ -23,7 +23,7 @@ def test_addPersistent(wpilib, getTable):
     tab = wpilib.shuffleboard.Shuffleboard.getTab("tacos")
     widget = tab.addPersistent("simplevalue", 1)
     # k, how to kick networktables?
-    #assert widget.getEntry().isPersistent()
+    # assert widget.getEntry().isPersistent()
```

Comment:
rats, did I forget to ask about this for the shuffleboard pr?

Reply:
lol. that should be a thing, unless you forgot to make `getEntry` a thing.